vms/platformvm/validators/manager.go

```diff
@@ -203,6 +209,9 @@ func (m *manager) makePrimaryNetworkValidatorSet(
 	currentValidators, ok := m.cfg.Validators.Get(constants.PrimaryNetworkID)
 	if !ok {
 		// This should never happen
+		m.log.Error(ErrMissingValidator.Error(),
```

Comment:
There is a similar check in `makeSubnetValidatorSet` can we add the logging there too?

Reply:
Also - `"missing validator"` is not the best error / log here.. `"missing valdiator set"` would be better.